cfg/pgrapher/common/tools.jsonnet

```diff
@@ -133,7 +133,8 @@ function(params)
 		long_padding: 1.5*wc.ms,
 	    },
             uses: [$.dft, fr, $.elec_resp, $.rc_resp, $.sys_resp],
-        } for plane in [0,1,2]], $.fields),
+        } for plane in [0,1,2]], $.fields[r])
+	for r in std.range(0,14) ],
```

Comment:
You can not change the return value nor introduce icarus specific quantities here.

Reply:
how do you recommend addressing this?

Reply:
I see already a

```
cfg/pgrapher/experiment/icarus/icarus_tools.jsonnet
```

which even has a `pirs` definition.  So, I suggest this would be a good place to further modify.

But, some care is needed to not necessarily step on that if other ICARUS jobs rely on the current, uniform set of `pirs`.